code/modules/fabrication/designs/_design.dm

```diff
@@ -11,6 +11,7 @@
 	var/max_amount = 1 // How many instances can be queued at once
 	var/list/required_technology
 	var/list/species_locked
+	var/handle_as_stack
```

Comment:
i kind of would like this to be renamed `pass_order_multiplier` or something tbh but like, you don't HAVE to do that